source/jormungandr/jormungandr/instance_manager.py

```diff
@@ -284,27 +296,34 @@ def get_regions(self, region_str=None, lon=None, lat=None, object_id=None, api='
             return [i.name for i in valid_instances]
 
     def get_instances(self, name=None, lon=None, lat=None, object_id=None, api='ALL'):
-        available_instances = []
+        if name and name not in self.instances:
```

Comment:
@kadhikari  I added this line to handle the 404 cases

Reply:
:+1: